logstash-core/lib/logstash/api/service.rb

```diff
@@ -58,15 +39,7 @@ def get(key)
       private
 
       def has_counters?
-        (["LogStash::Instrument::MetricType::Counter", "LogStash::Instrument::MetricType::Gauge"] - metric_types).empty?
-      end
-
-      def metric_types
-        types = []
-        @snapshot_rotation_mutex.synchronize do
-          types = @snapshot.metric_store.all.map { |t| t.class.to_s }
-        end
-        return types
+        true
```

Comment:
good catch, went I did the red green dance, I've tried to reduce the variables :)